include/integratorxx/quadratures/gausscheby2_mod.hpp

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <integratorxx/quadrature.hpp>
+
+namespace IntegratorXX {
+
+  /**
+   *  @brief Implementation of modified Gauss-Chebyshev quadrature of the second kind
+   *
+   *  This method eliminates the sqrt(1-x^2) weight from the Chebyshev
+   *  quadrature rule by a change of variables. The nodes and weights
+   *  are still analytic.
+   *
+   *  References:
+   *  Comput. Phys. Commun. 70, 271 (1992)
+   *  doi:10.1016/0010-4655(92)90192-2
+   *
+   *  J. Chem. Phys. 100, 6520–6534 (1994)
+   *  doi:10.1063/1.467061
+   *
+   *  @tparam PointType  Type describing the quadrature points  
+   *  @tparam WeightType Type describing the quadrature weights 
+   */
+
+  template <typename PointType, typename WeightType>
+  class GaussChebyshev1 : 
```

Comment:
Presumably you want to name this more appropriately.

Reply:
This is copypaste from `gausscheby2.hpp`
https://github.com/wavefunction91/IntegratorXX/blob/0c8737e3fe2e9ef58d7bbf0dc68c881e78c0383e/include/integratorxx/quadratures/gausscheby2.hpp#L9C1-L9C1

Reply:
Good catch, that's wrong too, should be `GaussChebyshev2`, we've just never used it in prod. I'm not sure what the nature of the "modification" in this case is, but something to the effect of `GaussChebyshev2Mod` should work, but I'm fine with anything that would be distinct + descriptive.

Reply:
Yes. The existing routines also don't say where the formulas come from.

This is something that is IMHO quite badly documented in all of the papers on DFT quadrature.